src/main/scala/EnumTypes.scala

```diff
@@ -21,21 +21,19 @@ object EnumTypes {
     case Saturn  extends Planet(5.688e+26, 6.0268e7)
     case Uranus  extends Planet(8.686e+25, 2.5559e7)
     case Neptune extends Planet(1.024e+26, 2.4746e7)
-  }
+  end Planet
 
-  def test: Unit = {
+  def test(): Unit =
     val emptyList = ListEnum.Empty
     val list = ListEnum.Cons(1, ListEnum.Cons(2, ListEnum.Cons(3, ListEnum.Empty)))
-    println(emptyList)
+    println("Example 1: \n"+emptyList)
     println(s"${list}\n")
 
-    def calculateEarthWeightOnPlanets(earthWeight: Double) = {
+    def calculateEarthWeightOnPlanets(earthWeight: Double) =
       val mass = earthWeight/Planet.Earth.surfaceGravity
-      for (p <- Planet.values)
+      for p <- Planet.values do
         println(s"Your weight on $p is ${p.surfaceWeight(mass)}")
-    }
 
+    println("Example 2:")
     calculateEarthWeightOnPlanets(80)
-  }
 
```

Comment:
For longer functions and classes it would be good to use the `end` marker.

```suggestion
  end test

```